src/server/game/Battlegrounds/BattlegroundQueue.cpp

```diff
@@ -963,6 +955,31 @@ void BattlegroundQueue::BattlegroundQueueUpdate(BattlegroundBracketId bracket_id
     }
 }
 
+uint32 BattlegroundQueue::GetPlayesCountInGroupsQueue(BattlegroundBracketId bracketId, BattlegroundQueueGroupTypes bgqueue)
+{
+    uint32 PlayesCount = 0;
+
+    for (auto itr : m_QueuedGroups[bracketId][bgqueue])
```

Comment:
Why is this a for-loop btw? Arent you accessing a single entry by bracketId and bgQueue?

Reply:
Ah, yes yes